airflow/www/app.py

```diff
@@ -66,8 +66,20 @@ def create_app(config=None, testing=False, app_name="Airflow"):
     flask_app = Flask(__name__)
     flask_app.secret_key = conf.get('webserver', 'SECRET_KEY')
 
-    session_lifetime_days = conf.getint('webserver', 'SESSION_LIFETIME_DAYS', fallback=30)
-    flask_app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(days=session_lifetime_days)
+    default_session_lifetime_minutes = 43200
+    if conf.has_option('webserver', 'SESSION_LIFETIME_MINUTES'):
+        session_lifetime_minutes = conf.getint('webserver', 'SESSION_LIFETIME_MINUTES',
+                                               fallback=default_session_lifetime_minutes)
+    else:
+        warnings.warn('`SESSION_LIFETIME_DAYS` option from `webserver` section has been '
```

Comment:
Instead printing a warning, would it be better to make it a "hard stopper"?

My concern is some people normally simply skip warning messages, in  which case, the system behaviour would be different from what they expect.

Reply:
e.g. say I'm still using `SESSION_LIFETIME_DAYS` in my configuration, with non-default value, say 2 days. Assuming I'm a person who's used to skipping warnings, then what would actually happen is the session lifetime is gonna be 43200 min, i.e. 30 days, not what I think I'm configuring (2 days).

If I missed anything, let me know please. Thanks.

Reply:
@XD-DENG @mik-laj Sorry I was busy. I agree with @XD-DENG.  We should stop app when we detect removed options in configuration.

Reply:
Cool, we are aligned then.

Take your time please for the implementation if you are busy. Thanks!

Reply:
@XD-DENG @mik-laj I've updated code